### PR TITLE
Tidy up Documentation to reflect current status

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

File: project-overview.md [hunk 1]
```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
...
-To ensure that the code backs up the design, reference support for multiple attestation tokens will be developed in the project. The current targets for this support are:
+To ensure that the code backs up the design, reference support for multiple attestation tokens are supported in the project. The following are currently supported:
 
-* EAT Tokens, with the ARM PSA Profile being a specific test set.
-* DICE Tokens. NB: the current delivery format for the 'token', being the full DICE Certificate chain is still under consultation. If you're planning any such protocol implementations, we would be very interested in discussing them.
+* EAT Tokens, with the ARM PSA Profiles (Profile 1 & Profile 2) being a specific test set.
+* ARM Confidential Computing Architecture (CCA) Token and its variant [Parsec-CCA](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-cca.md)
```
Comment (thomas-fossati) [inline project-overview.md:22]: _variant_ is probably not the right word (there's no variation), _composition_ maybe more correct

File: project-overview.md [hunk 2]
```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
...
+* [DICE Tokens](https://trustedcomputinggroup.org/work-groups/dice-architectures/) (note: this does not implement any specific DICE architecture). If you're planning any specific profile of DICE protocol implementations, we would be very interested in discussing them.
```
Comment (thomas-fossati) [inline project-overview.md:23]: not sure what is meant by "DICE architecture" in this context?  In theory we can do both TCG and Open.

File: project-overview.md [hunk 3]
```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
...
+* TPM based attestation tokens namely [Enactrust](https://enacttrust.com/)  & [Parsec-TPM](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-tpm.md)
```
Comment (thomas-fossati) [inline project-overview.md:24]: * typo "Enactrust"
* extra space before ampersand
* missing comma after "tokens"

**Review by thomas-fossati — CHANGES_REQUESTED** [review]

File: project-overview.md (hunk 2 above)
Comment (setrofim) [inline project-overview.md:23]:
```suggestion
* [DICE](https://trustedcomputinggroup.org/work-groups/dice-architectures/) based attestation tokens (note: this does not adhere to any specific DICE-based protocol). If you're planning any particular DICE implementations, we would be very interested in discussing them.
```

Should probably also remove the reference to "DICE Tokens" here -- DICE specifically doesn't define anything like a token format, so there no such thing as a "DICE Token". (just like there so no such thing as a "TPM Token").

For the same reason, the term "profile" isn't really applicable here.

File: project-overview.md (hunk 1 above)
Comment (yogeshbdeshpande) [reply project-overview.md:22]: A variant is another version of something. This is correct usage, as Parsec tpm is a variant of base CCA token

Comment (thomas-fossati) [reply project-overview.md:22]: hmm, the CCA token is reused as-is by Parsec CCA, so there is absolutely no variation.  It is a composition of CCA and another piece that's stacked on top.

Comment (yogeshbdeshpande) [reply project-overview.md]: ok, without being hung up on the term, I have made the change!

File: project-overview.md (hunk 2 above)
Comment (yogeshbdeshpande) [reply project-overview.md]: Have addressed the concern and corrected some text, please have a look

File: project-overview.md (hunk 3 above)
Comment (yogeshbdeshpande) [reply project-overview.md]: Done!

File: project-overview.md [hunk 4]
```diff
@@ -19,9 +19,9 @@ Many real world attestation solutions and potential extrapolations of their use
...
 
 * EAT Tokens, with the ARM PSA Profiles (Profile 1 & Profile 2) being a specific test set.
-* ARM Confidential Computing Architecture (CCA) Token and its variant [Parsec-CCA](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-cca.md)
-* [DICE Tokens](https://trustedcomputinggroup.org/work-groups/dice-architectures/) (note: this does not implement any specific DICE architecture). If you're planning any specific profile of DICE protocol implementations, we would be very interested in discussing them.
-* TPM based attestation tokens namely [Enactrust](https://enacttrust.com/)  & [Parsec-TPM](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-tpm.md)
+* ARM Confidential Computing Architecture (CCA) Token and its composition [Parsec-CCA](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-cca.md)
+* [DICE layered Evidence](https://trustedcomputinggroup.org/work-groups/dice-architectures/) (note: this does not adhere to any specific DICE-based protocol). If you're planning any particular of DICE protocol implementations, we would be very interested in discussing them.
```
Comment (yogeshbdeshpande) [inline project-overview.md:23]: This needs a bit of tidy up, need some adjustment : will do

File: project-overview.md [hunk 5]
```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
 ### Tokens
 
 Many real world attestation solutions and potential extrapolations of their use have been considered to ensure that the general logic within the components performs the correct set of operations to provide support for the maximum set of deployments. Developing understanding and modelling the necessary flows for their verification process has driven the Veraison design.
-To ensure that the code backs up the design, reference support for multiple attestation tokens will be developed in the project. The current targets for this support are:
+To ensure that the code backs up the design, reference support for multiple attestation tokens are supported in the project. The following are currently supported:
```
Comment (thomas-fossati) [inline project-overview.md:19]:
```suggestion
To ensure that the code backs up the design, reference support for multiple attestation evidence formats are supported in the project. The following are currently supported:
```

File: project-overview.md [hunk 6]
```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
...
 
-* EAT Tokens, with the ARM PSA Profile being a specific test set.
-* DICE Tokens. NB: the current delivery format for the 'token', being the full DICE Certificate chain is still under consultation. If you're planning any such protocol implementations, we would be very interested in discussing them.
+* EAT Tokens, with the ARM PSA Profiles (Profile 1 & Profile 2) being a specific test set.
```
Comment (thomas-fossati) [inline project-overview.md:21]:
```suggestion
* EAT Tokens, with the [ARM PSA Profiles](https://datatracker.ietf.org/doc/draft-tschofenig-rats-psa-token/) being a specific test set.
```

File: project-overview.md [hunk 7]
```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
...
+* ARM Confidential Computing Architecture (CCA) Token and its composition [Parsec-CCA](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-cca.md)
```
Comment (thomas-fossati) [inline project-overview.md:22]:
```suggestion
* ARM Confidential Computing Architecture (CCA) Token and its composition [Parsec-CCA](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-cca.md).
```

File: project-overview.md [hunk 8]
```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
...
+* [DICE layered Evidence](https://trustedcomputinggroup.org/work-groups/dice-architectures/) (note: this does not adhere to any specific DICE-based protocol). If you're planning any particular DICE protocol implementation, we would be very interested in discussing it.
```
Comment (thomas-fossati) [inline project-overview.md:23]:
```suggestion
* [DICE layered Evidence](https://trustedcomputinggroup.org/work-groups/dice-architectures/).  Note that this does not adhere to any specific DICE-based protocol.  If you're planning any particular DICE protocol implementation, we would be very interested in discussing it.
```

File: project-overview.md [hunk 9]
```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
...
+* TPM based attestation tokens, namely [Enacttrust](https://enacttrust.com/) & [Parsec-TPM](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-tpm.md)
```
Comment (thomas-fossati) [inline project-overview.md:24]:
```suggestion
* TPM-based attestation evidence formats, namely [EnactTrust](https://enacttrust.com/) & [Parsec-TPM](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-tpm.md).
```

File: project-overview.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (yogeshbdeshpande) [reply project-overview.md]: Yes, I had corrected it in the latest commit. Thanks.

**Review by thomas-fossati — COMMENTED** [review]
further tidy up suggestion inlined

File: project-overview.md (hunk 5 above)
Comment (yogeshbdeshpande) [reply project-overview.md:19]: The title is tokens for the section and we have used attestation tokens so I would prefer to retain this terminology!

File: project-overview.md (hunk 9 above)
Comment (yogeshbdeshpande) [reply project-overview.md:24]: again no need to change to Evidence formats

File: project-overview.md (hunk 8 above)
Comment (yogeshbdeshpande) [reply project-overview.md:23]: Again a editorial view of looking at the same thing, not much different from the text which is currently written. I would retain an explicit note as this does better on emphasis.

File: project-overview.md (hunk 5 above)
Comment (thomas-fossati) [reply project-overview.md:19]: as pointed out by Sergei in another review comment, some of these (namely DICE) are not strictly "tokens"

File: project-overview.md (hunk 8 above)
Comment (thomas-fossati) [reply project-overview.md:23]: I think the proposal reads a bit better.  Removing the parenthetical connects the two related sentences ("Note that [...]." and "If you are planning [...]) and makes the flow smoother.  Besides, "note" should be capitalised.

File: project-overview.md (hunk 9 above)
Comment (thomas-fossati) [reply project-overview.md:24]: this goes along with the other

File: project-overview.md (hunk 5 above)
Comment (yogeshbdeshpande) [reply project-overview.md:19]: This has been addressed in the titles and clarified on the individual specific bullets

**Review by SimonFrost-Arm — COMMENTED** [review]
Looks good. Perhaps the list of currently supported needs to be followed by something along the lines of: 
"Support for additional architectures is a matter of frequent discussion and we expect this list to expand" 
WDYT?

**Comment by yogeshbdeshpande** [comment]
> Looks good. Perhaps the list of currently supported needs to be followed by something along the lines of: "Support for additional architectures is a matter of frequent discussion and we expect this list to expand" WDYT?

Totally agree and I will add that! Thank you for the review

File: project-overview.md [hunk 10]
```diff
@@ -16,10 +16,15 @@ If the following resonates with any of your plans, if you would like to contribu
...
 
-* EAT Tokens, with the ARM PSA Profile being a specific test set.
-* DICE Tokens. NB: the current delivery format for the 'token', being the full DICE Certificate chain is still under consultation. If you're planning any such protocol implementations, we would be very interested in discussing them.
+*  EAT Tokens, with the [ARM PSA Profiles](https://datatracker.ietf.org/doc/draft-tschofenig-rats-psa-token/) being a specific test set.
+* ARM Confidential Computing Architecture (CCA) Token and its composition [Parsec-CCA](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-cca.md).
+* [DICE layered Evidence](https://trustedcomputinggroup.org/work-groups/dice-architectures/) (Note: this does not adhere to any specific DICE-based protocol). If you're planning any particular DICE protocol implementation, we would be very interested in discussing it.
+* TPM based attestation tokens, namely [Enacttrust](https://enacttrust.com/) & [Parsec-TPM](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-tpm.md).
```
Comment (thomas-fossati) [inline project-overview.md:24]: * moving the closing parenthesis to the right place
* correct capitalisation of EnactTrust

```suggestion
* [DICE layered Evidence](https://trustedcomputinggroup.org/work-groups/dice-architectures/) (Note: this does not adhere to any specific DICE-based protocol. If you're planning any particular DICE protocol implementation, we would be very interested in discussing it.)
* TPM based attestation tokens, namely [EnactTrust](https://enacttrust.com/) & [Parsec-TPM](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-tpm.md).
```

Comment (yogeshbdeshpande) [reply project-overview.md:24]: The parenthesis is really meant for the Note, not for the second statement. 
Thanks for the suggestion on the EnactTrust, will do it!

Comment (yogeshbdeshpande) [reply project-overview.md]: ok, thinking a bit more, it does not hurt to bring everything in the parenthesis, I will change this!

Comment (thomas-fossati) [reply project-overview.md]: The second sentence is actually part of the note.  See the original text please.

Comment (yogeshbdeshpande) [reply project-overview.md]: > The second sentence is actually part of the note. See the original text please.

yes, i have corrected it already! Thanks!

**Review by thomas-fossati — APPROVED** [review]
I still think in L19 "attestation tokens" should be "attestation evidence formats", but the rest LGTM

**Review by SimonFrost-Arm — APPROVED** [review]
LGTM

**Comment by yogeshbdeshpande** [comment]
@SimonFrost-Arm and @thomas-fossati and @setrofim 
Thank you all for the review!

**Pull request merged**